app/controllers/admin/accounting/transactions_controller.rb

```diff
@@ -6,4 +6,40 @@ def index
 
     initialize_transactions_grid
   end
+
+  def new
+  end
+
+  def create
+    @loan = Loan.find(transaction_params[:loan_id])
+    authorize @loan
+    @transaction = ::Accounting::Transaction.new(project: @loan)
+
+    begin
+      creator = ::Accounting::Quickbooks::TransactionCreator.new
+      creator.add_disbursement(
+        amount: transaction_params[:amount],
+        loan_id: transaction_params[:loan_id],
+        description: '',
+        memo: transaction_params[:private_note],
+        qb_bank_account_id:  transaction_params[:qb_bank_account_id],
+        organization: @loan.organization,
+      )
+
+      render_modal_partial
+    rescue => ex
+      flash.now[:error] = ex.message
+      render_modal_partial(status: 422)
```

Comment:
Because the quickbooks errors happen outside of the transaction object, simple form does not render any errors from quickbooks. Is there a better way to display these errors to the user?

Though if a QB error does happen, the form values are still saved.